packages/core/src/service/helpers/utils/get-user-channels-permissions.ts

```diff
@@ -14,6 +14,7 @@ export interface UserChannelPermissions {
 
 /**
  * Returns an array of Channels and permissions on those Channels for the given User.
+ * @deprecated See `RolePermissionResolverStrategy`
```

Comment:
Do we really need to deprecate this first? This helper isn't exported, so I would say it's ok to just mention in the changelog that you should use `configService.authOptions.rolePermissionResolverStrategy.resolvePermissions` from now on.